pkg/volume/local/local.go

```diff
@@ -107,6 +107,7 @@ func (plugin *localVolumePlugin) NewMounter(spec *volume.Spec, pod *v1.Pod, _ vo
 			mounter:    plugin.host.GetMounter(),
 			plugin:     plugin,
 			globalPath: volumeSource.Path,
+                        MetricsProvider: volume.NewMetricsDu(volumeSource.Path),
```

Comment:
NewMetricsDu calculates the volume usage and device free space by executing "du", NewMetricsStatFS calculates the used and available space by stat'ing and gathering filesystem info for the Volume path.